src/testdrive/src/action.rs

```diff
@@ -441,7 +441,10 @@ pub async fn build(cmds: Vec<PosCommand>, state: &State) -> Result<Vec<PosAction
                     "kafka-ingest" => Box::new(kafka::build_ingest(builtin).map_err(wrap_err)?),
                     "kafka-verify" => {
                         Box::new(kafka::build_verify(builtin, context.clone()).map_err(wrap_err)?)
-                    }
+                    },
+                    "kafka-verify-schema" => {
```

Comment:
tentatively put this under `kafka` rather than a more generic schema check action... the schemas get unique names each test run, and they're resolved by querying the `mz_kafka_sinks` table like the existing `kafka-verify` does